data/persons.txt

```diff
@@ -1850,3 +1850,201 @@ ship "Modified Ladybug"
+			"Down with tangents, up with overlaps."
+			"Don't be afraid of adding more contrast."
+			"If you want to learn how to create artwork, start with a donut."
+
+#		tibetiroka
+			"It's just the Steam CI, no need to panic."
+			"Did you check for nullptr?"
+			"		How do we indent after line breaks again?"
+			"Please put the includes in alphabetical order."
+			"Always keep three empty lines between functions."
+			"Why do aliens speak English?"
+
+#		bene-dictator
+			"Just a syntax review. Hold on, I have to rewrite this whole paragraph for you quickly. Delay the merge!"
+			"I was requested for spelling? Oh my goodness, this needs a rewrite..."
+			"One more review. Pretty please?"
+			"Just one more review before merging."
+			"Capitalise that proper noun. Use American English for consistency. For goodness' sake, that needs a comma!"
+			"Comma here. Comma doesn't go there. I think that one needs a comma. Hold on, what was a comma again?"
+			"Unidentified sentence appears to be lacking a comma. Very dangerous, and highly likely to incite multiple suggestions on where the comma should go. Calling backup reviewers."
+			"Commas are your greatest friends."
+			"Commas are your greatest enemies."
+
+#		roadrunner56
+			"More semicolons!"
+			"...I think I broke something."
+			"Canadian spelling fails me again."
+			"This is my last review, I swear."
+			"Looks like school's starting back up again. See you in a few months."
+			"Typo detected. Initiating weapons engagement."
+			"Code non-functional. Destroy them."
+
+#		EjoThims
+	phrase
+		word
+			"If you want your pull request not to be merged"
+			"If you prefer that your pull request never be merged"
+			"If you prefer that your pull request sits idle"
+			"If you want your pull request to be closed"
+		word
+			", "
+		word
+			"make sure you keep adding content to it"
+			"always change unrelated things"
+			"do not ever test it and don't add integration tests. Whatever those even are, they're apparently important"
+			"try your best to avoid respecting existing syntax and conventions"
+			"just forget about it for a while"
+			"use the forbidden word: 'realism'"
+			"bury the new feature in semi-relevant data changes"
+			"do not, under any circumstances, ever be willing to compromise"
+			"make sure to completely ignore what the reviewers tell you to do"
+			"skip over writing an issue about it first"
+		word
+			"!"
+			"."
+			". Trust me! It works."
+			". You shouldn't do those things, though! I'm just kidding."
+
+	ship "Waverider" "Proofreader's Pain"
+		"never disabled"
+		add attributes
+			"bunk" 4
+	ship "Waverider" "Art Critics' Regret"
+		"never disabled"
+		add attributes
+			"bunk" 2
+	ship "Waverider" "Code Herders' Curse"
+		"never disabled"
+		add attributes
+			"required crew" 2
+	ship "Waverider" "Choreographers' Tears"
+		"never disabled"
+		outfits
+			"Particle Cannon" 4
+			"S-970 Regenerator"
+			"Dwarf Core"
+			"LP576a Battery Pack"
```

Comment:
```suggestion
			"LP576a Battery Pack"
```
Why is this one different here? It's making it so the ship has negative space.

Reply:
In reference to the one mission to the Deep mission chain where "no one uses the LP576a"

Reply:
I guess it's just a matter of if we care about giving one of them an extra 60 outfit space for the sake of this joke. 

Personally I'd prefer if it was just the original, since very few people will know the reference to it and it's not outstated, that way there all more neatly consistent. It's a person ship sure, but I don't really think we need this one.

Reply:
Back to the same point on not overcomplicating stuff by having each ship having the exact number of reviewers, eh?

Reply:
Essentially.

Reply:
A few things to note:

1. How many people are going to end up scanning this ship (OK, a few are)?
2. How many people actually pay attention to what the battery is (the human batteries have too similar names for me)?
3. Is anyone going to get the joke if they do the other things?

Reply:
Yeah I'd just have it the same as the other ships.

Reply:
Has been addressed.